king_phisher/spf.py

```diff
@@ -208,11 +216,12 @@ class SenderPolicyFramework(object):
 	if an IP address is authorized to send messages on it's behalf. The exp
 	modifier defined in section 6.2 of the RFC is not supported.
 	"""
-	def __init__(self, ip, domain, sender=None):
+	def __init__(self, ip, domain, timeout, sender=None):
```

Comment:
Should probably be a keyword arg here as well. To keep them both synced with the same value, you should add a constant to the top of the file and document it with the others. Something like `DEFAULT_DNS_TIMEOUT = 10` and them `timeout=DEFAULT_DNS_TIMEOUT`.